app/node_modules

```diff
@@ -0,0 +1 @@
+../node_modules
```

Comment:
This will not work on Windows and has some other issues:
- Serving all your npm dependencies is probably not the best idea.
- This makes it harder to deploy the app, because you have to copy both `app/` and `node_modules/`.
- Due to the above, this increases the size of the deploy artifacts significantly.